src/main/resources/org/jenkins/ci/plugins/column/console/LastBuildColumn/column.jelly

```diff
@@ -29,7 +29,7 @@
         ${%ConsoleColumn.LastBuild.NotAvailable}
       </j:when>
       <j:otherwise>
-        <a href="job/${job.name}/lastBuild/console" title="${%ConsoleColumn.LastBuild.Title}">
+        <a href="${job.url}lastBuild/console" title="${%ConsoleColumn.LastBuild.Title}">
```

Comment:
Probably needs to be prefixed by `${rootURL}/` for this to work when the view is itself inside a folder.